firmware/boron-ins-fsm/src/imDoorSensor.cpp

```diff
@@ -232,4 +241,9 @@ int isDoorOpen(int doorStatus) {
  */
 int isDoorStatusUnknown(int doorStatus) {
     return (doorStatus == INITIAL_DOOR_STATUS);
+}
+
+// returns the consecutive times a heartbeat with an open door has been sent
+unsigned long getConsecutiveHeartbeatCount() {
+    return consecutiveDoorOpenHeartbeatCount;
```

Comment:
Question: if you're using consecutiveDoorOpenHeartbeatCount in src/stateMachine.cpp, then is there a reason you have this function in src/imDoorSensor.cpp? It may be redundant if the variable is global

Reply:
Otherwise, please call this function in src/stateMachine.cpp and don't have the extern mention in src/imDoorSensor.h to ensure that the variable is only modified in src/imDoorSensor.cpp

Reply:
this is a leftover function. it's not used at all. removed in latest push